robot_controllers/src/cartesian_twist.cpp

```diff
@@ -282,14 +304,21 @@ void CartesianTwistController::command(const geometry_msgs::Twist::ConstPtr& goa
     return;
   }
 
+  std::string frame;
+  frame = goal->header.frame_id;
+  if (frame.empty())
+  {
+    return;
```

Comment:
you should also be calling requestStop() here, otherwise you are still relying on the 0.5s timeout before the controller stops.